### ISSUE TITLE
Create-AlProjectFolderFromBcContainer requires credentials

### ISSUE THREAD

**Issue by ajkauffmann** [microsoft/navcontainerhelper#605]
The latest change to Create-AlProjectFolderFromBcContainer requires a $credential variable when not using baseline and version is 15.

```
Create-AlProjectFolderFromBcContainer : The variable '$credential' cannot be retrieved because it has not been set.
At C:\Users\ajkau\OneDrive\Documenten\PowerShell\CreateALWorkspace.ps1:19 char:1
+ Create-AlProjectFolderFromBcContainer -containerName bconlinev15 `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (credential:String) [Create-AlProjectFolderFromNavContainer], RuntimeException
    + FullyQualifiedErrorId : VariableIsUndefined,Create-AlProjectFolderFromNavContainer
```

I guess a parameter -Credential should be added
A workaround is to create a $credential variable yourself before calling the cmdlet.

**Comment by freddydk** [comment]
You are right.

**Comment by freddydk** [comment]
Shipped in 0.6.4.1
